src/webextension/icons/clear.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you run this through `svgo` to remove the unnecessary bits? Whatever settings @changecourse is using for the other SVGs should be fine.